doc/glossary.rst

```diff
@@ -127,7 +127,8 @@ reflect either technical terms or legacy ways of referring to Ceph systems.
 	Dashboard Plugin
 	Dashboard
                 A built-in web based monitoring and administration application to
-                Ceph Manager. Refer :doc:`/mgr/dashboard` for more details.
+                Ceph Manager. Refer :ref:`Ceph Manager Dashboard <mgr-dashboard-overview>`
```

Comment:
How about applying the following to `dashboard.rst`, to simplify the references?
```diff --git a/doc/mgr/dashboard.rst b/doc/mgr/dashboard.rst
index 2b653384f4..1b1a603c69 100644
--- a/doc/mgr/dashboard.rst
+++ b/doc/mgr/dashboard.rst
@@ -1,3 +1,5 @@
+.. _mgr-dashboard:
+
 Ceph Manager Dashboard
 ======================
 
```

This way, you can simply refer to `mgr-dashboard` without an alternative title to create a reference.

Reply:
Yes that is also possible. But I thought I could use the unused `mgr-dashboard-overview` label. So what do you suggest for the unused `mgr-dashboard-overview` label. Drop or Keep it?

Reply:
Sorry, wrong grep. Fixed now. Please check.